tests/test_http_response.py

```diff
@@ -14,9 +14,12 @@ def normalize_headers(self, headers):
         """
         return sorted(
             [
+                (b"server", b"daphne"),
```

Comment:
The case should be preserved as above

```suggestion
                (b"server", b"Daphne"),
```